app/src/main/java/io/apicurio/registry/rest/v2/GroupsResourceImpl.java

```diff
@@ -564,6 +565,22 @@ public Response getArtifactVersion(String groupId, String artifactId, String ver
         return builder.build();
     }
 
+    /**
+     * @see io.apicurio.registry.rest.v2.GroupsResource#deleteArtifactVersion(java.lang.String, java.lang.String, java.lang.String)
+     */
+    @Override
```

Comment:
Authorization annotation is missing. We thing it should be `AuthorizedLevel.Write`.

Reply:
Fixed